Frontend/ui-driver/src/Screens/ScheduledRideAcceptedScreen/View.purs

```diff
@@ -203,15 +187,17 @@ instructionView  push state  =
        textView $[
         height WRAP_CONTENT
         ,width WRAP_CONTENT
-        ,color Color.black900
+        ,color Color.black700
+        ,  fontWeight $ FontWeight 600
         ,textFromHtml $ getString BE_WITHIN_10KM_OF_PICKUP
       
       ]<>FontStyle.h3 TypoGraphy
       , textView $[
         height WRAP_CONTENT
         ,width WRAP_CONTENT
-        , color Color.black900
-        ,textFromHtml $ getString BEFORE_THE_RIDE_STARTS
+        , color Color.black700
+        ,  fontWeight $ FontWeight 600
+        ,textFromHtml $ getString  BEFORE_THE_RIDE_STARTS
       ]<>FontStyle.h3 TypoGraphy
```

Comment:
You can create separate font style for your use case with font weight as 600 and everything else similar to h3